### PR TITLE
copy Rt episode in middle of delays episodes

### LINKED ISSUE #18: make copy of `quantify-transmission` episode to tutorials-early
This suggest to have trial 01 visible on one website only
File in https://github.com/epiverse-trace/tutorials-middle/blob/main/episodes/quantify-transmissibility.Rmd

### LINKED ISSUE #41: remove the space before the colon ":"
> Remove the space before the colon ":"

_Originally posted by @Degoot-AM in https://github.com/epiverse-trace/tutorials-early/pull/37#discussion_r1548189936_

We can assess for homogeneity on this in all episodes

### PR DESCRIPTION
**Pull request by avallecam**
Fix #18 
Fix #41 

Commit history of file in https://github.com/epiverse-trace/tutorials-middle/blob/main/episodes/quantify-transmissibility.Rmd
review in https://github.com/epiverse-trace/tutorials-early/blob/md-outputs-PR-37/quantify-transmissibility.md
sync PR https://github.com/epiverse-trace/tutorials-middle/pull/28

### CODE REVIEW

File: episodes/quantify-transmissibility.Rmd [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+
+
+
+
+## Quantify geographical heterogeneity
+
+The outbreak data of the start of the COVID-19 pandemic from the United Kingdom from the R package `{incidence2}` includes the region in which the cases were recorded. To find regional estimates of the effective reproduction number and cases, we must format the data to have three columns:
+
++ `date` : the date,
++ `region` : the region,
```
Comment (Degoot-AM) [inline episodes/quantify-transmissibility.Rmd:430]: Remove the space before the colons ":"

File: episodes/quantify-transmissibility.Rmd [hunk 2]
```diff
@@ -0,0 +1,474 @@
...
+
+estimates_regional$summary$summarised_results$table
+
+estimates_regional$summary$plots$R
+```
+
+
+## Summary
+
+`EpiNow2` can be used to estimate transmission metrics from case data at the start of an outbreak. The reliability of these estimates depends on the quality of the data and appropriate choice of delay distributions. In the next tutorial we will learn how to make forecasts and investigate some of the additional inference options available in `EpiNow2`.
```
Comment (Degoot-AM) [inline episodes/quantify-transmissibility.Rmd:467]: Why you keep emphasizing the point `at the start of an outbreak`? I sought this package can be used any time in course of the outbreak period. Note that It calculated the effective (not the basic) reproduction number.

**Review by Degoot-AM — APPROVED** [review]

File: episodes/quantify-transmissibility.Rmd [hunk 3]
```diff
@@ -0,0 +1,474 @@
...
+```{r setup, echo = FALSE, warning = FALSE, message = FALSE}
+library(EpiNow2)
+library(ggplot2)
+withr::local_options(list(mc.cores = 4))
+```
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- How can I estimate the time-varying reproduction number ($Rt$) and growth rate from a time series of case data?
+- How can I quantify geographical heterogeneity in these transmission metrics?
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:16]:
```suggestion
- How can I quantify geographical heterogeneity from these transmission metrics? 
```

File: episodes/quantify-transmissibility.Rmd [hunk 4]
```diff
@@ -0,0 +1,474 @@
...
+
+- Learn how to estimate transmission metrics from a time series of case data using the R package `EpiNow2`
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: prereq
+
+## Prerequisites
+
+Learners should familiarise themselves with following concept dependencies before working through this tutorial:
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:31]:
```suggestion
Learners should familiarise themselves with following concepts before working through this tutorial: 
```

File: episodes/quantify-transmissibility.Rmd [hunk 5]
```diff
@@ -0,0 +1,474 @@
...
+
+But in an ongoing outbreak, the population does not remain entirely susceptible as those that recover from infection are typically immune. Moreover, there can be changes in behaviour or other factors that affect transmission. When we are interested in monitoring changes in transmission we are therefore more interested in the value of the **effective reproduction number**, $R_t$, the average number of cases caused by one infectious individual in the population at time $t$.
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+## Introduction
+
+Quantifying transmission metrics at the start of an outbreak can give important information on the strength of transmission (reproduction number) and the speed of transmission ([growth rate](../learners/reference.md#growth), doubling/halving time). To estimate these key metrics using case data we must account for delays between the date of infections and date of reported cases. In an outbreak situation, data are usually available on reported dates only, therefore we must use estimation methods to account for these delays when trying to understand changes in transmission over time.
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:53]:
```suggestion
The transmission intensity of an outbreak is quantified using two key metrics: the reproduction number, which informs on the strength of the transmission by indicating how many new cases are expected from each existing case; and the [growth rate](../learners/reference.md#growth), which informs on the speed of the transmission by indicating how rapidly the outbreak is spreading or declining (doubling/halving time) within a population. To estimate these key metrics using case data we must account for delays between the date of infections and date of reported cases. In an outbreak situation, data are usually available on reported dates only, therefore we must use estimation methods to account for these delays when trying to understand changes in transmission over time. 
```

File: episodes/quantify-transmissibility.Rmd [hunk 6]
```diff
@@ -0,0 +1,474 @@
...
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+## Introduction
+
+Quantifying transmission metrics at the start of an outbreak can give important information on the strength of transmission (reproduction number) and the speed of transmission ([growth rate](../learners/reference.md#growth), doubling/halving time). To estimate these key metrics using case data we must account for delays between the date of infections and date of reported cases. In an outbreak situation, data are usually available on reported dates only, therefore we must use estimation methods to account for these delays when trying to understand changes in transmission over time. 
+
+In the next tutorials we will focus on how to implement the functions in `{EpiNow2}` to estimate transmission metrics of case data. We will not cover the theoretical background of the models or inference framework, for details on these concepts see the [vignette](https://epiforecasts.io/EpiNow2/dev/articles/estimate_infections.html).
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:55]:
```suggestion
In the next tutorials we will focus on how to use the functions in `{EpiNow2}` to estimate transmission metrics of case data. We will not cover the theoretical background of the models or inference framework, for details on these concepts see the [vignette](https://epiforecasts.io/EpiNow2/dev/articles/estimate_infections.html).
```

File: episodes/quantify-transmissibility.Rmd [hunk 7]
```diff
@@ -0,0 +1,474 @@
...
+For more details on the distinction between speed and strength of transmission and implications for control, see [Dushoff & Park, 2021](https://royalsocietypublishing.org/doi/full/10.1098/rspb.2020.1556).
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:56]:
```suggestion

```

File: episodes/quantify-transmissibility.Rmd [hunk 8]
```diff
@@ -0,0 +1,474 @@
...
+The R package `EpiNow2` uses a [Bayesian inference](../learners/reference.md#bayesian) framework to estimate reproduction numbers and infection times based on reporting dates.
+
+In Bayesian inference, we use prior knowledge (prior distributions) with data (in a likelihood function) to find the posterior probability.
+
+<p class="text-center" style="background-color: white">Posterior probability $\propto$ likelihood $\times$ prior probability
+</p>
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::::::::::::::::::::::::::::::::::: instructor
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:71]: no, but we can try to add titles to them to link or redirect to them. Is that the feature you request?

File: episodes/quantify-transmissibility.Rmd [hunk 9]
```diff
@@ -0,0 +1,474 @@
...
+
+We refer to the Prior probability distribution and the [Posterior probability](https://en.wikipedia.org/wiki/Posterior_probability) distribution.
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:73]:
```suggestion
We refer to the prior probability distribution and the [posterior probability](https://en.wikipedia.org/wiki/Posterior_probability) distribution.
```

File: episodes/quantify-transmissibility.Rmd [hunk 10]
```diff
@@ -0,0 +1,474 @@
...
+```
+
+To use the data, we must format the data to have two columns:
+
++ `date` : the date (as a date object see `?is.Date()`),
++ `confirm` : number of confirmed cases on that date.
+
+Let's use `{tidyr}` and `incidence2::incidence()` for this:
+
+```{r, warning = FALSE, message = FALSE}
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:109]: agree for consistency. At least for packages out of the main ones that belong to the main objective of the episode.

File: episodes/quantify-transmissibility.Rmd [hunk 11]
```diff
@@ -0,0 +1,474 @@
...
+
+<p class="text-center" style="background-color: aliceblue">**time from infection to symptom onset** (the [incubation period](../learners/reference.md#incubation)) + **time from symptom onset to case notification** (the reporting time)
+.</p>
+
+The delay distribution for each of these processes can either estimated from data or obtained from the literature. We can express uncertainty about what the correct parameters of the distributions by assuming the distributions have **fixed** parameters or whether they have **variable** parameters. To understand the difference between **fixed** and **variable** distributions, let's consider the incubation period. 
+
+::::::::::::::::::::::::::::::::::::: callout
+### Delays and data
+The number of delays and type of delay is a flexible input that depends on the data. The examples below highlight how the delays can be specified for different data sources:
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:152]:
```suggestion
The number of delays and type of delay are a flexible input that depend on the data. The examples below highlight how the delays can be specified for different data sources:
```

File: episodes/quantify-transmissibility.Rmd [hunk 12]
```diff
@@ -0,0 +1,474 @@
...
+</center>
+
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+
+#### Incubation period distribution 
+
+The distribution of incubation period can usually be obtained from the literature. The package `{epiparameter}` contains a library of epidemiological parameters for different diseases obtained from the literature.
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:171]:
```suggestion
The distribution of incubation period for many diseases can usually be obtained from the literature. The package `{epiparameter}` contains a library of epidemiological parameters for different diseases obtained from the literature. 
```

File: episodes/quantify-transmissibility.Rmd (hunk 2 above)
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:467]:
```suggestion
`EpiNow2` can be used to estimate transmission metrics from case data at any time in the course of an outbreak. The reliability of these estimates depends on the quality of the data and appropriate choice of delay distributions. In the next tutorial we will learn how to make forecasts and investigate some of the additional inference options available in `EpiNow2`. 
```

File: episodes/quantify-transmissibility.Rmd [hunk 13]
```diff
@@ -0,0 +1,474 @@
...
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: prereq
+
+## Prerequisites
+
+Learners should familiarise themselves with following concept dependencies before working through this tutorial: 
+
+**Statistics** : probability distributions, principle of Bayesian analysis. 
+
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:34]: added in #41

File: episodes/quantify-transmissibility.Rmd [hunk 14]
```diff
@@ -0,0 +1,474 @@
...
+**Epidemic theory** : Effective reproduction number.
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:35]: added in #41

File: episodes/quantify-transmissibility.Rmd (hunk 10 above)
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:108]: Added in #42

File: episodes/quantify-transmissibility.Rmd [hunk 15]
```diff
@@ -0,0 +1,474 @@
...
+To illustrate the functions of `EpiNow2` we will use outbreak data of the start of the COVID-19 pandemic from the United Kingdom. The data are available in the R package `{incidence2}`. 
+
+```{r}
+dplyr::as_tibble(incidence2::covidregionaldataUK)
+```
+
+To use the data, we must format the data to have two columns:
+
++ `date` : the date (as a date object see `?is.Date()`),
++ `confirm` : number of confirmed cases on that date.
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:104]: added in #41

File: episodes/quantify-transmissibility.Rmd (hunk 1 above)
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:429]: added in #41

File: episodes/quantify-transmissibility.Rmd (hunk 10 above)
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:108]: solved in https://github.com/epiverse-trace/tutorials-early/pull/37/commits/beba32c9311e102b64285c4344e0b3dd44288804

File: episodes/quantify-transmissibility.Rmd [hunk 16]
```diff
@@ -0,0 +1,474 @@
...
+
+To illustrate the functions of `EpiNow2` we will use outbreak data of the start of the COVID-19 pandemic from the United Kingdom. The data are available in the R package `{incidence2}`. 
+
+```{r}
+dplyr::as_tibble(incidence2::covidregionaldataUK)
+```
+
+To use the data, we must format the data to have two columns:
+
++ `date` : the date (as a date object see `?is.Date()`),
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd]: solved in https://github.com/epiverse-trace/tutorials-early/pull/37/commits/beba32c9311e102b64285c4344e0b3dd44288804

File: episodes/quantify-transmissibility.Rmd (hunk 15 above)
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd]: solved in https://github.com/epiverse-trace/tutorials-early/pull/37/commits/beba32c9311e102b64285c4344e0b3dd44288804

File: episodes/quantify-transmissibility.Rmd (hunk 14 above)
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd]: solved in https://github.com/epiverse-trace/tutorials-early/pull/37/commits/beba32c9311e102b64285c4344e0b3dd44288804

File: episodes/quantify-transmissibility.Rmd (hunk 13 above)
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/quantify-transmissibility.Rmd:34]: solved in https://github.com/epiverse-trace/tutorials-early/pull/37/commits/beba32c9311e102b64285c4344e0b3dd44288804

**Pull request merged**